### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.13.11
+
+* Implemented new scoring report.
+* Improved handling of tar-files
+* Better path handling on windows.
```
Comment (mit-mit) [inline CHANGELOG.md:5]: s/windows/Windows/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 0.13.11
+
+* Implemented new scoring report.
+* Improved handling of tar-files
```
Comment (isoos) [inline CHANGELOG.md:4]: s/tar-files/tar files/ ?

Comment (sigurdm) [reply CHANGELOG.md]: I rephrased slightly.

File: CHANGELOG.md (hunk 1 above)
Comment (sigurdm) [reply CHANGELOG.md]: Done

**Pull request merged**